CONTRIBUTING.md

```diff
@@ -18,13 +18,36 @@ Welcome! We're so excited that you're considering contributing to the official G
 ---
 
 ## Commit your update
+Please follow our project commit guide lines below:
+- The first line is the title of the message,
+it should be no longer than 63 characters.
+- The second line should be left empty.
+- The remaining consist of the body of the message,
+they should be no longer than 72 characters each.
+try to be descriptive as you can while keeping it simple.
+
+- Example:
+Adding an awesome feature.
+
+Adding feature which:
+1. makes cookies.
+2. makes you get A+ in all courses.
+3. create dank memes.
+in order to use each of the feature above just go to page ****,
+write the line "make_cookies()" where ever you want and walla, 
+cookies will appear.
```

Comment:
For me, this example wasn't clear. 
First of all, I would write it inside a code block: `Example: this is a code block.`
And it is a little bit too long for me. The purpose here is to show how to write a good commit message.